{
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
        "ID": "CVE-2019-3900",
        "ASSIGNER": "secalert@redhat.com",
        "STATE": "PUBLIC"
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "vendor_name": "Red Hat",
                    "product": {
                        "product_data": [
                            {
                                "product_name": "Kernel",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "affects up to and including v5.1-rc6"
                                        }
                                    ]
                                }
                            }
                        ]
                    }
                }
            ]
        }
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "CWE-835"
                    }
                ]
            }
        ]
    },
    "references": {
        "reference_data": [
            {
                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-3900",
                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-3900",
                "refsource": "CONFIRM"
            },
            {
                "url": "https://www.spinics.net/lists/kernel/msg3111012.html",
                "name": "https://www.spinics.net/lists/kernel/msg3111012.html",
                "refsource": "CONFIRM"
            },
            {
                "refsource": "BID",
                "name": "108076",
                "url": "http://www.securityfocus.com/bid/108076"
            },
            {
                "refsource": "FEDORA",
                "name": "FEDORA-2019-87d807d7cb",
                "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/TOFNJA5NNVXQ6AV6KGZB677JIVXAMJHT/"
            },
            {
                "refsource": "FEDORA",
                "name": "FEDORA-2019-8219efa9f6",
                "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/AYTZH6QCNITK7353S6RCRT2PQHZSDPXD/"
            },
            {
                "refsource": "FEDORA",
                "name": "FEDORA-2019-a6cd583a8d",
                "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/RI3WXXM5URTZSR3RVEKO6MDXDFIKTZ5R/"
            },
            {
                "refsource": "CONFIRM",
                "name": "https://security.netapp.com/advisory/ntap-20190517-0005/",
                "url": "https://security.netapp.com/advisory/ntap-20190517-0005/"
            },
            {
                "refsource": "REDHAT",
                "name": "RHSA-2019:1973",
                "url": "https://access.redhat.com/errata/RHSA-2019:1973"
            }
        ]
    },
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "An infinite loop issue was found in the vhost_net kernel module in Linux Kernel up to and including v5.1-rc6, while handling incoming packets in handle_rx(). It could occur if one end sends packets faster than the other end can process them. A guest user, maybe remote one, could use this flaw to stall the vhost_net kernel thread, resulting in a DoS scenario."
            }
        ]
    },
    "impact": {
        "cvss": [
            [
                {
                    "vectorString": "6.3/CVSS:3.0/AV:N/AC:H/PR:L/UI:N/S:C/C:N/I:N/A:H",
                    "version": "3.0"
                }
            ]
        ]
    }
}